conan/tools/cmake/utils.py

```diff
@@ -1,5 +1,41 @@
+from conan.errors import ConanException
+
 
 def is_multi_configuration(generator):
     if not generator:
         return False
     return "Visual" in generator or "Xcode" in generator or "Multi-Config" in generator
+
+
+def parse_extra_variable(source, key, value):
```

Comment:
Not a fan of moving this to a utils file, but didn't find a better suited place to share between CMakeToolchain and CMakeConfigDeps, happy to hear alternatives